bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/services/amp/AmpTransformer.java

```diff
@@ -219,6 +228,27 @@ private String getJsContent() {
         return formatJsOutput(output.toString());
     }
 
+    private Resource getHeadlibResourceSuperType(ResourceResolver resolver, String resourceType, String headLibRelPath) {
+        Resource resource = Utils.resolveResource(resolver, resourceType);
```

Comment:
so you've tested this on a publisher as an anonymous user and this works?

Reply:
As I told you in the other comment, I'm constantly receiving _null_ with this approach even in author, that is why I'm asking if I'm missing some configuration.

Reply:
Ah I was just looking at the file changed - and the comment above disappeared. I'll reply to your other comment.